channel-sdk/src/main/java/com/zello/channel/sdk/InformationalError.kt

```diff
@@ -0,0 +1,25 @@
+package com.zello.channel.sdk
+
+import org.json.JSONObject
+
+/**
+ *
+ */
+open class InformationalError internal constructor(val errorMessage: String) {
```

Comment:
Not saying you should change it, but I am curious why you chose to implement this as a standalone class and pass it through rather than extending `Throwable` and using try/catch blocks in the places where you might encounter it. Is it just a personal preference or is there some other advantage you gain from doing it that way?

Reply:
So far it's used to report errors encountered _e.g._ when the server sends an invalid message. So it would really have no business interrupting any running code.

I'm definitely more used to Objective-C's "avoid using exceptions" style of programming than Kotlin/Java's exception based error handling though, so I may miss places where it's appropriate to use them.